src/application-navigation/ApplicationNavigationActionsContext.js

```diff
@@ -0,0 +1,42 @@
+import { createContext } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * The ApplicationNavigationActionsContext can be used by Layout implementations to define
```

Comment:
You could make the comments more ApplicationNavigation specific vs. the general "Layout" terminology from v2.

Reply:
Updated.